### PR TITLE
G-mode Crateria Kihunter Room

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

**Review by osse101 — COMMENTED** [review]
An upward GMode Immobile entry seems possible to me.  Crabs are global so supers will knock them down.

File: tech.json [hunk 1]
```diff
@@ -898,6 +898,15 @@
...
                 }
               ]
+            },
+            {
+              "name": "canDownwardGModeSetup",
+              "requires": [ "canEnterRMode" ],
+              "note": [
+                "Ability to setup an R-mode or G-mode through a downward door.",
+                "Samus needs to be in a standing or walking position with no vertical speed on the first frame in the next room, while taking damage through the transition.",
+                "This is typically done using quicksand, clipping into the floor, or using a thawing frozen wall crawler."
```
Comment (osse101) [inline tech.json:908]: Pretty sure any type of frozen enemy could work.  Covern, Atomic, a Gamet would be worth testing.

File: region/crateria/east.json [hunk 2]
```diff
@@ -347,6 +387,25 @@
...
+                {
+                  "name": "Get Hit by Thawing Sciser",
+                  "notable": false,
+                  "requires": [
+                    "canTrickyUseFrozenEnemies",
+                    "canDownwardGModeSetup",
+                    {"ammo": {"type": "Super", "count": 1}}
+                  ],
+                  "note": [
+                    "Run from the ledge into the open doorway while hitting the frozen Crab as it thaws and the door transition simultaneously.",
```
Comment (osse101) [inline region/crateria/east.json:403]: More precisely, if I understand it right: You hit the frozen crab, then the transition, and the crab unfreezes during the transition.

**Review by osse101 — COMMENTED** [review]
👍

File: tech.json (hunk 1 above)
Comment (kjbranch) [reply tech.json:908]: True. I would think any enemy would work, probably anything besides a crawler will end up notable tho. I am particularly curious about the covern/atomic though

**Pull request merged**